presto-rcfile/src/test/java/io/prestosql/rcfile/RcFileTester.java

```diff
@@ -654,6 +655,7 @@ private static DataSize writeRcFileColumnNew(File outputFile, Format format, Com
     {
         OutputStreamSliceOutput output = new OutputStreamSliceOutput(new FileOutputStream(outputFile));
         AircompressorCodecFactory codecFactory = new AircompressorCodecFactory(new HadoopCodecFactory(RcFileTester.class.getClassLoader()));
+        @SuppressWarnings("resource")
```

Comment:
Can you remove this and instead make sure the writer gets always closed?

Reply:
Same, will revert